MainProject/UABCompany.cs

```diff
@@ -0,0 +1,21 @@
+﻿namespace MainProject
+{
+    public class UABCompany : Company
+    {
+        private string _ownerName;
+
+        public UABCompany(string name, int year, string vatNumber, string ownerName) : base(name, year, vatNumber)
+        {
+            this._ownerName = ownerName;
```

Comment:
Please remove key word "this" where it's redundant

Reply:
Fixed